test/Object/rlexe.xml

```diff
@@ -453,19 +453,31 @@
   <test>
     <default>
       <files>ObjectSpread_Simple.js</files>
-      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectSpread</compile-flags>
+      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectRestSpread</compile-flags>
     </default>
   </test>
   <test>
     <default>
       <files>ObjectSpread_JIT.js</files>
-      <compile-flags>-args summary -endargs -ES2018ObjectSpread -bgjit -maxinterpretcount:1 -off:simplejit</compile-flags>
+      <compile-flags>-args summary -endargs -ES2018ObjectRestSpread -bgjit- -maxinterpretcount:1 -off:simplejit</compile-flags>
     </default>
   </test>
   <test>
     <default>
       <files>ObjectSpread_Limits.js</files>
-      <compile-flags>-args summary -endargs -ES2018ObjectSpread</compile-flags>
+      <compile-flags>-args summary -endargs -ES2018ObjectRestSpread</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>ObjectRest_Simple.js</files>
+      <compile-flags>-args summary -endargs -NoNative -ES2018ObjectRestSpread -forceserialized</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>ObjectRest_JIT.js</files>
+      <compile-flags>-args summary -endargs -ES2018ObjectRestSpread -bgjit- -maxinterpretcount:1 -off:simplejit -forceserialized</compile-flags>
```

Comment:
>-bgjit- -maxinterpretcount:1 -off:simplejit [](start = 68, length = 43)

you don't need to give this - rl.exe will make sure to pass this on interpreted and dynapogo mode. Can you verify that as well? after running the test you can see the logs at logs\x64_debug\interpreted\rl.full.log

Reply:
It seems rl.exe passes -mic and -bgjit-, but not -off:simplejit (it instead sets -maxSimpleJitRunCount:1) so I've left in -off:simplejit.

---
In reply to: [205548586](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r205548586) [](ancestors = 205548586)